### ISSUE TITLE
Leviathan Primary Cannon

### ISSUE THREAD

**Issue by HellDragon-HK** [GreatEmerald/UT3Vehicles#142]
- GAME PLAY RELATED
  - [ ] In ONS-Urban Blue Power Core at 100% was destroyed with a SINGLE main cannon shot, is that normal?  I was on Instant Action and had 0 bots in the game so there was no interference damage....wasn't able to replicate this so guess it was a lucky shot, on most other attempts it took 2 shots with the first bringing the core down to 21% and on one attempt it dropped to 45%
  - [X] Main cannon takes about 5 seconds to turn half way around on UT3 and only 2 with the port

 - VISUAL EFFECTS RELATED
  - [ ] The Levi has it's energy pulled back into it's cannon barrel as it charges then throws it in a huge explosion, there are too many visual effects here to list accurately at this time

**Comment by GreatEmerald** [comment]
The first issue is being tackled by #103. Without the damage scaling it would indeed deal incorrect damage.

**Comment by HellDragon-HK** [comment]
What about factoring in the missing 500 hp on the core, can't really compare games because of that

**Comment by GreatEmerald** [comment]
Indeed. You could estimate what the damage should be to destroy the core, though.
